website/docs/cloud-docs/api-docs/policy-sets.mdx

```diff
@@ -765,6 +835,54 @@ curl \
   https://app.terraform.io/api/v2/policy-sets/polset-3yVQZvHzf5j3WRJ1/relationships/policies
 ```
 
+## Detach a Policy Set from projects
+
+`DELETE /policy-sets/:id/relationships/projects`
+
+| Parameter | Description                                                                                      |
+| --------- | ------------------------------------------------------------------------------------------------ |
+| `:id`     | The ID of the policy set to attach to projects. Use the "List Policy Sets" endpoint to find IDs. |
```

Comment:
Typo - The ID of the policy set to detach from projects.

Reply:
Good catch! It's been updated.